utility/src/main/java/org/oppia/util/parser/SvgSoftwareLayerSetter.kt

```diff
@@ -0,0 +1,33 @@
+package org.oppia.util.parser
+
+import android.graphics.drawable.PictureDrawable
+import com.bumptech.glide.load.DataSource
+import com.bumptech.glide.load.engine.GlideException
+import com.bumptech.glide.request.RequestListener
+import com.bumptech.glide.request.target.Target
+
+/**
+ * Listener which updates the {@link ImageView} to be software rendered, because {@link
```

Comment:
How does it do that? It seems from the implementation that the resource will never be ready, so it's not clear to me how it prevents hardware rendering.

Reply:
SVG format can not be hard decoding, it is soft decoding ImageView set at onResourceReady. I am also not so much aware of this. Need to dig in deep to understand.